conf/logback.xml

```diff
@@ -2,12 +2,10 @@
     <contextName>atom-workshop</contextName>
 
     <appender name="FILE" class="ch.qos.logback.core.rolling.RollingFileAppender">
-        <file>${application.home}/logs/application.log</file>
+        <file>logs/application.log</file>
 
         <rollingPolicy class="ch.qos.logback.core.rolling.FixedWindowRollingPolicy">
-            <fileNamePattern>
-                ${application.home}/logs/application.log.%i
-            </fileNamePattern>
+            <fileNamePattern>logs/application.log.%i</fileNamePattern>
```

Comment:
This change was prompted by this error in the logs:

> Jan 26, 2024 @ 12:06:34.865	12:06:32,373 |-ERROR in ch.qos.logback.core.rolling.RollingFileAppender[FILE] - openFile(application.home_IS_UNDEFINED/logs/application.log,true) call failed. java.io.FileNotFoundException: application.home_IS_UNDEFINED/logs/application.log (No such file or directory)

Whatever `${application.home}` was, it's not available anymore! With the updated settings ([copied from Ophan](https://github.com/guardian/ophan/blob/2ecb80da40915ce097e4a1de0e29b352b396323e/dashboard/conf/logback.xml#L16-L19)) the error message is gone, and we still get log messages in the https://logs.gutools.co.uk/ ELK, where it matters.

See also https://stackoverflow.com/a/47331903, though I don't think we really needed the fallback.